constants.py

```diff
@@ -238,6 +238,18 @@ def with_variables(self, variables: JsonType) -> GQLOperation:
 
 
 GQL_OPERATIONS: dict[str, GQLOperation] = {
+    # retuns PlaybackAccessToken_Template, for fix 2024/5
+    "PlaybackAccessToken": GQLOperation(
+        "PlaybackAccessToken",
+        "3093517e37e4f4cb48906155bcd894150aef92617939236d2508f3375ab732ce",
+        variables={
+            "isLive": True,
+            "login": "carbinecarlito",
```

Comment:
Placeholders are supposed to be empty, or at least contain something that is clear about being supposed to be replaced, like an ellipsis.
```suggestion
            "login": "...",
```